composer.json

```diff
@@ -41,7 +41,9 @@
         "ext-geoip": "Enabling the geoip extension allows you to use the MaxMindProvider.",
         "geoip/geoip": "If you are going to use the MaxMindBinaryProvider (conflict with geoip extension).",
         "geoip2/geoip2": "If you are going to use the GeoIP2DatabaseProvider.",
-        "symfony/stopwatch": "If you want to use the TimedGeocoder"
+        "symfony/stopwatch": "If you want to use the TimedGeocoder",
+        "ext-simplexml": "If you want to use HostIpXml",
+        "ext-json": "If you want to use HostIp"
```

Comment:
Please don't touch the main `composer.json` file.
Use `composer.json` file from provider.

Reply:
it's a "suggest" section - done it similar to `geoip2/geoip2`
composer.json from a provider will have a require